### ISSUE TITLE
Question about validation

### ISSUE THREAD

**Issue by ZhangYu1ing** [tianweiy/CenterPoint-KITTI#6]
Hi,

I have some questions regarding the validation set. 

1. As the code shown in the kitti_dataset.yaml, the 'kitti_infos_val.pkl' is regarded as the test set. I wonder the reason why is it set like this. 
<img width="315" alt="Screen Shot 2021-06-10 at 11 19 36 AM" src="https://user-images.githubusercontent.com/37301722/121499502-bf566e00-c9dd-11eb-8c24-8d69bf8040d6.png">

2. Then, whether there is a validation step in this algorithm? If the answer is yes, would you point out the validation is completed in which file?

3. And where did you split the training data and the validation data?

4. In your opinion and experience, what is the number of epochs that could let CenterPoint full trained on KITTI? Or in another word, why did you choose to use 80 epochs to train the CenterPoint? 

Thanks a lot~

**Comment by tianweiy** [comment]
sorry for the late reply. 

1234 <- all of these are directly followed from openpcdet. Please ask questions in their repo. 

The following are my personal ideas.

1. because we don't get annotation for the test set (you need to submit to the server)

2. I think there is one after you finish the training. You need to check the code yourself though.

3. please ask in openpcdet repo. 

4. it is the default setting

**Comment by ZhangYu1ing** [comment]
Okay. We see. Thanks a lot.
